online-shop/src/app/product-detail/product-detail.component.scss

```diff
@@ -0,0 +1,174 @@
+@import url('https://fonts.googleapis.com/css?family=Fjalla+One|Montserrat:300,400,700,800|Open+Sans:300');
```

Comment:
**suggestion:** fonts should either be downloaded and kept inside the `/assets` folder or if you like be referenced inside the `index.html`